stubs/third_party_taint/xss_sinks.pysa

```diff
@@ -6,6 +6,9 @@ def markupsafe.Markup.__new__(cls, base: TaintSink[XSS], encoding, errors) : ...
 @SkipObscure
 def markupsafe._native.escape(s: TaintInTaintOut[Via[escape_html]]): ...
 
+# Markup library used by Flask
+def markupsafe.Markup.escape(s: TaintInTaintOut[Via[escape_html]]): ...
```

Comment:
Two tweaks we should make here:
1) Adding `@SkipObscure`. This is something we just released, and is documented here: https://pyre-check.org/docs/pysa-advanced#obscure-models . We want `SkipObscure` as a temporary workaround for this issue that's preventing us from getting `always-via:escape_html` (you'll notice in your test we're only getting `via:escape_html`)
2) Adding `LocalReturn`. This is so we actually propagate taint from `s` to the return value, despite the `SkipObscure` we're adding. Documented here: https://pyre-check.org/docs/pysa-basics#taint-propagation

```suggestion
@SkipObscure
def markupsafe.Markup.escape(s: TaintInTaintOut[LocalReturn, Via[escape_html]]): ...
```